access/ms-teams/bot.go

```diff
@@ -61,7 +72,7 @@ func NewBot(c msapi.Config, clusterName, webProxyAddr string) (*Bot, error) {
 		Config:      c,
 		graphClient: msapi.NewGraphClient(c),
 		botClient:   msapi.NewBotFrameworkClient(c),
-		users:       make(map[string]UserData),
+		recipients:  make(map[string]RecipientData),
```

Comment:
I'm assuming that the fundamental point of this change is that access requests can now be sent to a channel or user, so just calling the field `users ` is now misleading?